Slash/Utility/Comments/Comments.pm

```diff
@@ -1198,47 +820,22 @@ sub printComments {
 		cid		=> $cid,
 		pid		=> $pid,
 		cc		=> $cc,
-		lcp		=> $lcp,
```

Comment:
leave in for now.

we may want to have pagination as a admin switchable settubg